sdk/__init__.py

```diff
@@ -89,12 +89,9 @@ def get_batch(self, id: int, load_results: bool = False) -> Batch:
             Batch: the batch stored in the PCS database.
         """
 
-        batch_rsp = self._client._get_batch(id)
+        batch_rsp, jobs_rsp = self._client._get_batch(id, fetch_results=load_results)
```

Comment:
Should we standardize `load_results` and `fetch_results` or are they any different ?